ingest/expression_files/dense_ingestor.py

```diff
@@ -28,6 +28,8 @@
 class DenseIngestor(GeneExpression, IngestFiles):
     # ToDo SCP-2635
     ALLOWED_FILE_TYPES = ["text/csv", "text/plain", "text/tab-separated-values"]
+    GENE_BATCH_SIZE = 5
```

Comment:
I don't think we should be batching by gene. The gene documents are very small. Batching by gene will make more calls to the database than necessary. If this is kept the amount of genes should be significantly larger than the amount of data arrays.

Reply:
+1